content/docs/tasks/traffic-management/egress/egress-sample.txt

```diff
@@ -0,0 +1,23 @@
+# $snippet SNIP1
```

Comment:
A test program can't just run this example, because only the first line is a command. The remaining lines are expected output.

Reply:
So we need a mechanism to capture test output and put into the site? Kind of makes my head hurt, not sure how to wire that up.